src/main/java/at/ac/tuwien/kr/alpha/config/CommandLineParser.java

```diff
@@ -68,6 +68,12 @@
 			.desc("provide the asp program as a string").build();
 	private static final Option OPT_LITERATE = Option.builder("l").longOpt("literate")
 			.desc("enable literate programming mode (default: " + InputConfig.DEFAULT_LITERATE + ")").build();
+	private static final Option OPT_WRITE_PREPROCESSED = Option.builder("pp").longOpt("write-preproc").hasArg(true).argName("target")
+			.desc("write the internal program that is passed into the solver after transformations to a file").build();
+	private static final Option OPT_WRITE_DEPGRAPH = Option.builder("dG").longOpt("depgraph").hasArg(true).argName("target")
```

Comment:
not really, I thought it would be the more intuitive variant, especially "dg" being quite close to "dbg" (debug checks). But I changed it to lower-case now, so the options look a bit more uniform.